src/Services/Air/AirFormat.js

```diff
@@ -86,6 +86,17 @@ function getBaggageInfo(info) {
 }
 
 function formatSegment(segment) {
+  const isCodeshare = (Object.prototype.toString.call(segment['air:CodeshareInfo']) === '[object Object]');
+  let codeshareInfo = {};
+
+  if (isCodeshare) {
+    codeshareInfo = {
+      operatingCarrier: segment['air:CodeshareInfo'].OperatingCarrier,
+      operatingCarrierName: segment['air:CodeshareInfo']._,
```

Comment:
Will it be the full name of carrier or what?

Reply:
This will be full name of carrier -ish.